packages/core/chrome/core-chrome-browser/src/project_navigation.ts

```diff
@@ -71,13 +78,16 @@ export interface ChromeProjectNavigation {
    * The navigation tree representation of the side bar navigation.
    */
   navigationTree: ChromeProjectNavigationNode[];
+  /**
+   * The flattened navigation tree representation of the side bar navigation.
+   * This is optional, if not provided it will be computed from the navigationTree
+   */
+  navigationTreeFlattened?: Record<string, ChromeProjectNavigationNode>;
 }
 
 /** @public */
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface SideNavCompProps {
-  // TODO: provide the Chrome state to the component through props
-  // e.g. "navTree", "activeRoute", "recentItems"...
+  activeNodes: ChromeProjectNavigationNode[][];
```

Comment:
is `ChromeProjectNavigationNode.title` what is rendered in the sidenav? Can I just use it for a breadcrumb?

Reply:
Yes that's what you need to use for the breadcrumb. It could happen (not in the current project though) that a team decides to use a `ReactNode` to render what's in the side nav. In that case they would have to provide a `title` string for the breadcrumb. In any case it would appear under `title` on the node.